packages/experimental/NativeButton/macos/MSFButtonViewManager.m

```diff
@@ -0,0 +1,30 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+#import "RCTBridgeModule.h"
+
+@import FluentUI;
+
+@implementation RCTConvert (MSFButtonAdditions)
+RCT_ENUM_CONVERTER(MSFButtonStyle, (@{
+	@"primary": @(MSFButtonStylePrimary),
+	@"secondary": @(MSFButtonStyleSecondary),
+	@"acrylic": @(MSFButtonStyleAcrylic),
+	@"borderless": @(MSFButtonStyleBorderless),
+}), MSFButtonStylePrimary, integerValue);
+
+@end
+
+@interface RCT_EXTERN_MODULE(MSFButtonViewManager, RCTViewManager)
```

Comment:
I personally like one view manager per platform better, but could we combine the two viewManagers into one and use ifdefs like what what @amgleitman has been doing in react-native-macos and community modules?

Reply:
I find it easier to read  and less prone to bugs with two separate viewManagers.